CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ Please follow [the Keep a Changelog standard](https://keepachangelog.com/en/1.0.
 
 ## [Unreleased]
 
+## [3.0.2]
```

Comment:
Please, add this note into 2.x.x too and bump the version there too within the changelog so that the changelog is truthy.

Reply:
Im confused, should it be the same for 2.x.x as well?

Reply:
I thought about this a bit more and I think it's ok to omit it here :)

Sorry, can't think straight when I'm sick